react-form-controls/my-app/src/RegistrationFormControlled.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+export default function RegistrationFormControlled() {
+  const [isUsername, setUsername] = useState('');
+  const [isPassword, setPassword] = useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    console.log(isUsername, isPassword);
+  }
+  function handleChange(event) {
+    let isUsername = event.target.isUsername;
```

Comment:
`isUsername` is already declared as a state value. If we want to change its value, we need to use the `setUsername` function to update it.

Also, to make this function more reusable, use `event.target.value` instead. I encourage you to `console.log(event.target)` and take a look at all the properties and methods!